README.md

```diff
@@ -69,23 +69,56 @@ After implementing filtering in your application, this is the way to filter via
 
 Sorting query results is controlled via the _sort_by_ query parameter. The _sort_by_ parameter is available for both REST API and GraphQL requests.
 
-The syntax for the _sort_by_ parameter supports:
+The _sort_by_ parameter specifies which attribute name to sort the results by, and may include a sort order of ascending _asc_ or descending _desc_. The default behavior when no sorting order is specified is to sort by ascending order.
 
-- a single string representing the attribute name to sort by which may be followed by :asc or :desc
-  - **attribute**   (_default order is ascending_)
-  - **attribute:asc** (_ascending order_)
-  - **attribute:desc** (_descending order_)
-- an array of strings of the above syntax
+The syntax for the _sort_by_ parameter is as follows:
+
+- One or more object keys representing the attribute name(s) to sort by which may be assigned the **asc** or **desc** value for the sort order.
+
+  - [**attribute**]   (_default order is ascending_)
+  - [**attribute**]=**asc** (_ascending order_)
+  - [**attribute**]=**desc** (_descending order_)
 
 ##### Sort_by Examples:
 
-- GET /api/v1.0/sources?sort_by=name
-- GET /api/v1.0/vms?sort_by[]=power_state\&sort_by[]=memory:desc
+- GET /api/v2.0/sources?sort_by[name]
+- GET /api/v2.0/vms?sort_by[power_state]&sort_by[memory]=desc
 
 | Query Parameter | Ruby Client Parameter | GraphQL Parameter |
 | --------------- | --------------------- | ----------------- |
-| "?sort_by=name" | { :sort_by => "name" } | sort_by: "name" |
-| "?sort_by[]=power_state\&sort_by[]=memory:desc" | { :sort_by => ["power_state", "memory:desc"] } |  sort_by: ["power_state, "memory:desc"] |
+| "?sort_by[name]" | { :sort_by => { :name => nil } } | sort_by: { name: null } |
+| "?sort_by[name]=asc" | { :sort_by => { :name => "asc" } } | sort_by: { name: "asc" } |
+| "?sort_by[power_state]&sort_by[memory]=desc" | { :sort_by => { :power_state => nil, :memory => "desc" } } | sort_by: { power_state: null, memory: "desc" } |
+
+#### Filtering and Sorting by Association attribute
+
+Requests can also be filtered by assocation attribute and sorted by association attribute and count in addition to the direct attribute specified as in the above examples.
+
+Single level association can be specified as follows:
+
+##### Filter by association attribute:
+
+| Query Parameter | Ruby Client Parameter <br> **GraphQL filter: Parameter** |
```

Comment:
Should these tables be 3 columns like the one above?

Reply:
I think that will look better, lemme give that a try 👍

Reply:
ok, pushed an update. Decided to normalize all tables with the 3 columns, much nicer to see. Thanks!!